### DISCUSSION TITLE
You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

### DISCUSSION THREAD

**Discussion by Max-Rohrhirsch** [community/community#36453] [GitHub Education]
Hi, I tried to get Github education. I've  uploaded the document that I'm in school but when I clicked on "Process my application" I got this answer: 
Your application cannot be reviewed until you fix the following:
- You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
- Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

But I already edited my full name and bio.

**Comment by max0n4ik** [comment]
I submitted 9 applications and all the same result, have you solved this problem somehow?

**Reply by Jarzynski** [reply]
Have you solved this problem? I have the same question no matter what I submit. I dont know what I didn't fill in. I want to sent mails to GitHub service but I dont know how

**Reply by max0n4ik** [reply]
the problem was solved by itself. I just sent everything the same way the next day and they accepted my application

**Reply by Jarzynski** [reply]
me too. I found I had suddenly passed this certification after establishing my first repo.

**Comment by hqh-cell** [comment]
I had the same problem

**Reply by max0n4ik** [reply]
i just waited, following day they agree my request.

**Reply by hqh-cell** [reply]
All right，thanks bro.

**Comment by Yapibe** [comment]
i had the same problem, waited for the next day and tried signing up while in my university so it could detect my location and it worked

**Comment by katrinmisel** [comment]
It has something to do with having access to your location as @Yapibe said. I updated my bio and my name and I was still getting the same error for 2 days straight with Chrome but when I tried to upload my proof of enrollment over the phone browser just now it worked instantly. Hope it works for you!

**Reply by MaoChengzhi** [reply]
thanks! I've been trying on PC for several hours but failed. Then I made it on ipad instantly

**Comment by heywji** [comment]
Same as so

**Comment by Jarzynski** [comment]
Actually, it wasn't until I uploaded some scripts to the library that I was approved. Maybe he thought I'm a robot.....

**Comment by VimukthiniAttanayaka** [comment]
I succeeded on my ninth attempt.
